lib/segment/src/index/hnsw_index/graph_links.rs

```diff
@@ -269,6 +283,40 @@ impl GraphLinksConverter {
     }
 }
 
+pub fn convert_to_compressed(path: &Path, m: usize, m0: usize) -> OperationResult<()> {
+    let start = std::time::Instant::now();
+
+    let links = GraphLinksMmap::load_from_file(path)?;
+    if links.info.compressed {
+        return Ok(());
+    }
+
+    let edges = (0..links.num_points())
+        .map(|point_id| {
+            let num_levels = links.point_level(point_id as PointOffsetType) + 1;
+            (0..num_levels)
+                .map(|level| links.links_vec(point_id as PointOffsetType, level))
+                .collect::<Vec<_>>()
+        })
+        .collect();
+    drop(links);
+    let mut converter = GraphLinksConverter::new(edges, true, m, m0);
+
+    let original_size = path.metadata()?.len();
+    converter.save_as(path)?;
+    let new_size = path.metadata()?.len();
+
+    log::info!(
```

Comment:
I propose to change log level to `debug` because HNSW graph logs have `debug` severity